webextensions/api/tabs.json

```diff
@@ -607,6 +607,25 @@
               }
             }
           },
+          "attention": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": true
+                },
+                "firefox_android": "mirror",
```

Comment:
In practice not supported on Android. The property exists but is always false.

Note: `tabs.query` needs an entry with the same version number, for the option `queryInfo.attention`, supported on Firefox 63+, and not supported at all on Android.